akka-remote/src/main/scala/akka/remote/artery/Codecs.scala

```diff
@@ -364,6 +420,19 @@ private[remote] class Decoder(
         }
       }
 
+      private def applyIncomingInstruments(decoded: InboundEnvelope, headerBuilder: HeaderBuilder): Unit = {
+        if (headerBuilder.flag(EnvelopeBuffer.MetadataPresentFlag)) {
+          val length = instruments.length
+          if (length == 0) {
+            val metaMetadataEnvelope = MetadataMapRendering.parse(headerBuilder)
+            log.warning("Incoming message envelope contains metadata for instruments: {}, " +
```

Comment:
I think this should be on debug level (with `if (log.isDebugEnabled)`), otherwise this can bring down the receiving system due to too much logging